wagtail/contrib/forms/models.py

```diff
@@ -121,6 +122,15 @@ class AbstractFormField(Orderable):
         FieldPanel("default_value", classname="formbuilder-default"),
     ]
 
+    api_fields = [
+        APIField("label"),
```

Comment:
I will add clean_name here - it would probably come in handy for generating forms clientside and be a more stable reference to the field over time.